example/storybook/stories/components/composites/Menu/MenuPositions.tsx

```diff
@@ -27,8 +27,6 @@ export function Example() {
       <Select
         selectedValue={position}
         mx={{ base: 0, md: 'auto' }}
-        accessibilityLabel="Select your favorite programming language"
```

Comment:
Instead of removing this, can we add a message like `Select a position`?

Reply:
I think it will be a bit odd if the `Menu` component itself doesn’t have a `accessibilityLabel` while the `Select` component has one. The example is for `Menu`, and `Select` is just there to help demonstrate `Menu` props.

Reply:
But Select should also convey what it does or what it contains right?

Reply:
I think you are right. I fixed it, and also found two more cases, fixed them as well.